assets/src/apps/authoring/components/Flowchart/rules/create-cata-choice-rules.ts

```diff
@@ -142,13 +142,14 @@ export const generateCATAChoiceRules = (
     return generateAllCorrectWorkflow(correct, [], disableAction, blankCondition);
   }
 
-  return generateThreeTryWorkflow(
+  return generateMaxTryWorkflow(
     correct,
     incorrect,
     commonErrorConditionsFeedback,
     setCorrect,
     blankCondition,
     disableAction,
+    { maxAttempt: screen?.content?.custom?.maxAttempt || '2' },
```

Comment:
I thought the default was 3, why '2' everywhere?

Reply:
That's how it has been in simple authoring. For maxOrLaterTry which was the default max 3 attempt in the existing behavior, the code was like this: 

```
const maxOrLaterTry = (value: string): ICondition => ({
  fact: 'session.attemptNumber',
  operator: 'greaterThan',
  value: '2',
  type: 1,
  id: guid(),
});
```
hence, I just replaced it with the user enter attempt and default to what it was before.